asteroids_code/BoilerPlate/player.hpp

```diff
@@ -1,14 +1,22 @@
 #pragma once
 
+#include <vector>
+
 #include "Vector2.hpp"
+#include "mathUtilities.hpp"
 
-const float move_speed = 5.0;
-const float rotation_angle = 4.0;
+const float move_speed = 5.0f;
+const float rotation_angle = 3.0f;
 class Player {
 
 	//members
+	MathUtilities conv;
```

Comment:
Un-intuitive object names. Why is this declared as local when its scope is the whole class.